docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-for-kubernetes-environments.md

```diff
@@ -33,22 +33,21 @@ Traces will be enhanced with Kubernetes metadata, similarly to the logs and metr
 
 ## Installing Sumo Logic Tracing on Kubernetes
 
-Installation is almost the same as for the official [SumoLogic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection), except a tracing flag (`sumologic.traces.enabled=true`) needs to be enabled. The process follows using a Helm chart to set all required components. It will automatically download and configure [OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which will collect, process, and export telemetry data to Sumo Logic.
+Installation is the same as for the official [SumoLogic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection). The process follows using a Helm chart to set all required components. It will automatically download and configure [OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which will collect, process, and export telemetry data to Sumo Logic.
 
 In the following installation steps, we use the release name `collection` and the namespace name `sumologic`. You can use any names you want, however, you'll need to adjust your installation commands to use your names since these names impact the OpenTelemetry Collector endpoint name.
 
+**NOTE:** In case of upgrade from Sumo Logic Kubernetes Collection `v2.x` to `v3.x` please see [Sumo Logic Kubernetes Collection Migration Guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/v3-migration-doc.md).
```

Comment:
```suggestion
**NOTE:** In case of upgrade from Sumo Logic Kubernetes Collection `v2.x` to `v3.x` please see [Sumo Logic Kubernetes Collection Migration Guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/v3-migration-doc.md).
```
```suggestion
:::note
 In case of upgrade from Sumo Logic Kubernetes Collection `v2.x` to `v3.x` please see [Sumo Logic Kubernetes Collection Migration Guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/v3-migration-doc.md).
::: 
```